<launch>
  <!-- <arg name="serial_port" default="/dev/ttyTHS0" /> -->
  <arg name="config_path" value="$(find tank_sdk)/config/config.yaml"/>
  <arg name="serial_port" default="/dev/ttyUSB0" />
  <arg name="serial_baudrate" default="115200" />
  <arg name="tank_id" default="0" />


  <!-- drivers for tianracer and tianbot are combined -->
  <node pkg="tank_sdk" type="tank_sdk_node" name="tank_$(arg tank_id)" output="screen">
    <param name="serial_port" type="string" value="$(arg serial_port)"/>
    <param name="serial_baudrate" type="int" value="$(arg serial_baudrate)"/>
    <param name="path" type="string" value="$(arg config_path)"/>
    <rosparam file="$(arg config_path)" command="load"/>
  </node>

</launch>
